clients/drcachesim/common/memtrace_stream.h

```diff
@@ -167,6 +167,40 @@ class memtrace_stream_t {
     {
         return -1;
     }
+
+    /**
+     * Returns a unique identifier for the current workload.  This might be an
+     * ordinal from the list of active workloads, or some other identifier.
```

Comment:
Should we return a PID instead? More meaningful than an ordinal and still unique enough (not guaranteed unique but we can live with that IMHO).

Reply:
Edit:
oh maybe that's what your comment here is saying "or some other identifier". Worth adding "like the PID"?

Reply:
For internal use we are using a UUID here.  For general use, there is already a pid in the memref_t struct, so having it here doesn't seem to add much.  The pid can be duplicated so a guaranteed-unique value was the idea here: UUID internally; ordinal externally.

Reply:
I added "This is guaranteed to
     * be unique among all inputs, unlike the process and thread identifiers in
     * #memref_t.".